services/slam/builtin/builtin.go

```diff
@@ -357,10 +357,14 @@ func configureCameras(ctx context.Context, svcConfig *AttrConfig, deps registry.
 				return "", nil, errors.Wrap(err, "error getting camera properties for slam service")
 			}
 
-			_, ok = props.DistortionParams.(*transform.BrownConrady)
+			brownConrady, ok := props.DistortionParams.(*transform.BrownConrady)
 			if !ok {
 				return "", nil, errors.New("error getting distortion_parameters for slam service, only BrownConrady distortion parameters are supported")
 			}
+			if err := brownConrady.CheckValid(); err != nil {
```

Comment:
Currently its impossible for this case to happen as `brownConrady` will never be nil if it was able to be case from DistortionParams, however if new validations are added to `brownConrady` in the future, this ensures we won't boot slam with invalid `brownConrady` params.